models/elghosh2024eucaim/metadata.yaml

```diff
@@ -0,0 +1,39 @@
+title: EUCAIM's Hyper-Ontology (Core Layer)
+acronym: 
+issued: 2024
+modified: 
+contributor:
+ - https://dblp.org/pid/199/9510.html
+ - https://dblp.org/pid/118/0942.html
+ - https://dblp.org/pid/43/1967.html
+ - https://dblp.org/pid/d/ChristelDanielLeBozec.html
+ - https://dblp.org/pid/96/1080.html
+ - https://dblp.org/pid/12/11202.html
+ - https://dblp.org/pid/83/4811.html
+ - https://dblp.org/pid/47/2102.html
+ - https://dblp.org/pid/20/3693.html
+keyword:
+ - oncology
+theme: Class R – Medicine
+editorialNote: The Core Layer of EUCAIM's Hyper-Ontology aims to ground an application ontology for the oncology domain on mCODE Ontological Conceptual Model, maintaining semantic interoperability.
+ontologyType:
+ - core
+ - application
+language: en
+designedForTask:
+ - conceptual clarification
+ - interoperability
+ - ontological analysis
+context:
+ - research
+ - industry
+source:
+ - https://www.utwente.nl/en/eemcs/fois2024/resources/papers/el-ghosh-et-al-towards-semantic-interoperability-among-heterogeneous-cancer-data-models-using-a-layered-modular-hyper-ontology.pdf
+ - https://www.utwente.nl/en/eemcs/fois2024/resources/papers/el-ghosh-et-al-grounding-a-hyper-ontology-on-mcode-ontological-conceptual-model-and-foundational-ontologies-for-semantic-interoperability-in-the-oncology-domain.pdf
+representationStyle:
+ - ontouml
+landingPage:
+ - https://catalogue.eucaim.cancerimage.eu/#/ 
```

Comment:
This does not seem to be the landing page of the ontology.

Reply:
URL removed.